tests/TestFuture.re

```diff
@@ -118,11 +118,10 @@ describe("Future", () => {
 
 
 describe("Future Belt.Result", () => {
+  let ((>>=), (<$>)) = Future.((>>=), (<$>));
 
   test("mapOk", () => {
-    Belt.Result.Ok("two")
-    |. Future.value
-    |. Future.mapOk(s => s ++ "!")
+    (Belt.Result.Ok("two") |. Future.value <$> (s => s ++ "!"))
```

Comment:
If we're going to provide the infix operators they should come with their own test and not replace the standard tests.  Tests are documentation, and having the infix operators in their own place would induce much less confusion.